### ISSUE TITLE
Incorrect app headers generated if platform requires something besides a single value

### ISSUE THREAD

**Issue by imclerran** [imclerran/roc-start#12]
Three out of four of the platforms listed in roc-start currently only require a single value from the application. For these platforms it is okay to generate a header like:
```roc
app [main] { pf: platform "https://..." }
```

However not all platforms follow this API structure. For example the `kingfisher` platform requires `[main, Model]`. 

`roc-start` does not currently have a solution to correctly generate the header for platforms which require something besides just `main`, and headers for `kingfisher` are currently not correct.

**Comment by imclerran** [comment]
A possible solution to this could be to update the format for a repository entry to something like:
```roc
 { repo: Str, owner: Str, alias: Str, platform: Bool, requires: Str }
 ```
 The `requires` field could then be something like: `requires: "main"`, or `requires: "main, Model"`.
 
The downside to this approach is it introduces a field which would be unused for packages. This could either be left blank, or packages and platforms could be split into two separate repositories, since this needs to be done on the `roc-start` application side anyway.

**Comment by imclerran** [comment]
Closed with PR #16
